### PR TITLE
Add IPv6 upstream server settings

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following (please check boxes, eg [X] - no spaces) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

_{replace this text with a number from 1 to 10, with 1 being not familiar, and 10 being very familiar}_

---

Implement full support for IPv6 upstream DNS servers.
This PR includes a bugfix that prevents correct displaying if more than 12 DNS servers are selected.

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

**Comment by PromoFaux** [comment]
Approved

**Comment by PromoFaux** [comment]
Looks nicer!

Approved

File: scripts/pi-hole/php/savesettings.php [hunk 1]
```diff
@@ -86,19 +86,36 @@ function readStaticLeasesFile()
...
-			"8.26.56.26" => "Comodo (Primary)",
-			"84.200.69.80" => "DNS.WATCH (Primary)",
-			"8.8.4.4" => "Google (Secondary)",
-			"208.67.220.220" => "OpenDNS (Secondary)",
-			"4.2.2.2" => "Level3 (Secondary)",
-			"199.85.127.10" => "Norton (Secondary)",
-			"8.20.247.20" => "Comodo (Secondary)",
-        	"84.200.70.40" => "DNS.WATCH (Secondary)",
+			"Google" => ["v4p" => "8.8.8.8","v4s" => "8.8.4.4", "v6p" => "2001:4860:4860:0:0:0:0:8888", "v6s" => "2001:4860:4860:0:0:0:0:8844"],
+			"OpenDNS" => ["v4p" => "208.67.222.222", "v4s" => "208.67.220.220"],
```
Comment (Skittels0) [inline scripts/pi-hole/php/savesettings.php]: [OpenDNS also has IPv6 servers.](https://www.opendns.com/about/innovations/ipv6/) Is there any reason why they are not implemented?

Comment (PromoFaux) [inline scripts/pi-hole/php/savesettings.php]: Probably no reason other than lack of knowing about them :)

Comment (Skittels0) [inline scripts/pi-hole/php/savesettings.php]: Would you give me the honor of making the PR? :)

Comment (DL6ER) [inline scripts/pi-hole/php/savesettings.php]: Go ahead!

**Pull request merged**